server/master/src/test/java/org/apache/accumulo/master/tableOps/ImportTableTest.java

```diff
@@ -38,21 +38,20 @@ public void testTabletDir() {
     iti.tableId = "5";
 
     // Different volumes with different paths
-    String[] tableDirs = new String[] {"hdfs://nn1:8020/apps/accumulo1/tables",
-        "hdfs://nn2:8020/applications/accumulo/tables"};
+    String[] volumes = {"hdfs://nn1:8020/apps/accumulo1", "hdfs://nn2:8020/applications/accumulo"};
```

Comment:
Accumulo 1.9 still only requires Java 1.7. The above won't compile with Java 1.7. When we build with Maven, we ensure we require at least 1.8, but this may cause compilation failures in an IDE when the IDE itself assumes 1.7 compliance.

Reply:
@ctubbsii thanks for pointing this out - fixed.